news/2020-09-13-project-loved-september-2020.md

```diff
@@ -0,0 +1,245 @@
+
+[Aural Vampire - Shounanzoku -cannibal coast-](https://osu.ppy.sh/beatmapsets/546972#mania) by [Wh1teh](https://osu.ppy.sh/users/2640467)\
+138 BPM, 3:10 | \[4K\] 4.94★
+
+We all know Wh1teh for his many popular and high quality charts, many of which have already been moved into Loved, with notable entrants including [TAKECORE OF YOURSELF](https://osu.ppy.sh/beatmapsets/480479#mania/1026050), [LA MD](https://osu.ppy.sh/beatmapsets/455656#mania/976620), and [Eighto](https://osu.ppy.sh/beatmapsets/584698#mania/1237015). Yet again we are seeing a chart of theirs nominated into the Loved Section in *Shounanzoku -cannibal coast-*, a chart that focuses primarily on dumping the vocals and layering the percussion to create quick, but relatively comfortable and satisfying jumpstream gameplay. The consistent nature of this representation makes it a joy to play through and with few breaks, something to keep players interested throughout, and can serve well as a stamina and consistency tester.  The satisfaction and testing ability of this chart has garnered a decent amount of support, and combining these qualities makes it an excellent nominee for the Loved section.
+
+[![](/wiki/shared/news/2020-09-13-project-loved-september-2020/mania/3-99-glooms.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Sta - 99 Glooms](https://osu.ppy.sh/beatmapsets/961775#mania) by [Ticle](https://osu.ppy.sh/users/11602148)\
+185 BPM, 2:36 | \[4K\] 5.29★
+
+A familiar face in the Loved section, Ticle returns to voting with *99 Glooms*. Ticle has made a name for himself as a very expressive mapper, taking much advantage of flowy patterns and long note releases. *99 Glooms* is a prime example of Ticle's signature style, filled with tricky patterns that one might almost describe as slippery or smooth, which is quite reflective of the music at many points. The map pushes the player into a state of focus and concentration with the immense usage of mini-LNs, which have to be released very quickly and precisely, and the fine utilization of staggered releases throughout the map; the usage of long notes with varying lengths also creates a tension that drives the player to pay closer attention. Ticle is also shown to be fairly creative and detailed in the manners in which particular sounds are expressed -- the map often leverages short, light, and quick bursts, which articulate on the lighter nature of the instruments like the piano and guitar, and there are multiple instances towards the end where the long notes which correspond a particular synth are seen to gradually get longer to represent a crescendo.\
+Overall, *99 Glooms* is a great experience for a wide range of players, not only due to how tricky it can get even for the most experienced of long note enthusiasts, but also because of the musical detail that is expressed. It's about time that this map got its chance to shine in the Loved section.
```

Comment:
```suggestion

Overall, *99 Glooms* is a great experience for a wide range of players, not only due to how tricky it can get even for the most experienced of long note enthusiasts, but also because of the musical detail that is expressed. It's about time that this map got its chance to shine in the Loved section.
```